apis/r/R/RcppExports.R

```diff
@@ -13,6 +13,46 @@ writeArrayFromArrow <- function(uri, naap, nasp, arraytype = "", config = NULL,
     invisible(.Call(`_tiledbsoma_writeArrayFromArrow`, uri, naap, nasp, arraytype, config, tsvec))
 }
 
+#' Create a group
+#' @param uri The array URI
+#' @param ctxxp An external pointer to the SOMAContext wrapper
+#' @export
+c_group_create <- function(uri, type, ctxxp, timestamp = NULL) {
+    invisible(.Call(`_tiledbsoma_c_group_create`, uri, type, ctxxp, timestamp))
+}
```

Comment:
Do we need this exported?

Reply:
Good catch!  That is copypasta.  None of the c_group_* should need to be user-facing.  I will clean that up.

Reply:
Done!  Let's see if CI still likes it.